src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+The Secret Management Service is a secure and scalable way to store and manage your secrets. It provides a centralized location for storing and managing secrets, such as API keys to reduce fragmentation and complexity across different services in New Relic. 
+You can store any string value as a secret, and call them programatically during runtime.
+
+Note that there's no data boundary for storing secrets; all secrets reside in US in a global DB enclave.
+
+Use the [NerdGraph API](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) to interact with the Secret Management Service. This tutorial will guide you through the process of creating, updating, and deleting secrets using the Secret Management Service.
+
+## Key features
+
+* **Centralized Repository**: Store all your confidential keys and credentials in one secure location.
+* **Secure Management**: Enhanced security features to protect your secrets.
+* **Scalability**: Designed to handle the growing needs of enterprises.
+
+## Requirements[#review-requirements]
+ 
+* You need a [New Relic account](https://newrelic.com/signup), and with that account, you can access your API user key that you need to include with queries and mutations.
+
+* User type and assigned roles can affect your NerdGraph permissions. For more details, see [Factors affecting access](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/#user-permissions).
+
+## Mutation operations
+
+To enable users to interact with the Secrets Management Service, following are the exposed four NerdGraph mutations:
+
+
+### Create a secret[#create-secret]
+
+The `secretsManagementCreateSecret` mutation allows you to add new secrets to the Secrets Management Service.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `alias` (String, optional): An alias for the secret.
+* `entityGuid` (String, optional): The entity GUID associated with the secret.
+* `key` (String, required): The key for the secret.
+* `namespace` (String, required): The namespace to which the secret belongs.
+* `value` (SecureValue, required): The value of the secret.
+
+#### Response
+
+* `createdAt` (String): The time the secret was created.
+* `updatedAt` (String): The time the secret was last updated.
+* `key` (String): The key of the created secret.
+* `latestVersion` (String): The version of the created secret.
+* `tags` (Array): Metadata associated with the secret.
+
+#### Sample query:
+
+```graphql
+mutation {
+  secretsManagementCreateSecret(
+    accountId: 12345678
+    key: "test_user_key"
+    value: "1234567890"
+    alias: ""
+    entityGuid: ""
+    namespace: ""
```

Comment:
Isn't namespace a required attribute? We should include meaningful samples in our docs. If we don't have one, let's get it from the PM/Eng. 

Ideally, we should provide working samples, which users can copy and execute.